src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -488,6 +488,11 @@ export default class ListPeopleAndRoles extends Mixins(CommonMixin) {
   .dropdown-action {
     border-left: 1px solid $gray3;
   }
+
+  // Override section container to show error bar
+  .section-container {
+    padding: 1.25rem 1.875rem 2.5rem;
```

Comment:
`section-container` is a global class. You should not override its padding.

Instead you should update the padding of whatever object is inside the section.

Reply:
Correct me if I'm wrong about the scoped style. I thought if the style is scoped, then `.people-roles-content .section-container` only makes changes inside the current component (`ListPeopleAndRoles`), even if `.section-container` is global.

Reply:
Yes, that is how scoped style works.

However, all "section containers" should have the same whitespace definition. What you are trying to do here is add whitespace, but it may belong on an inner element instead.

If you _really_ want to set a new CSS property here, use a different classname. Or use the [Vuetify spacing helpers](https://vuetifyjs.com/en/styles/spacing/).